src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -44,6 +47,61 @@ private ScannedDocumentsHelper() {
             .collect(toList());
     }
 
+    public static void setExceptionRecordIdToScannedDocuments(
+        ExceptionRecord exceptionRecord,
+        CaseUpdateDetails caseDetails
+    ) {
+        @SuppressWarnings("unchecked")
+        var caseData = (Map<String, Object>) caseDetails.caseData;
+        List<ScannedDocument> scannedDocuments = getScannedDocuments(caseData);
+
+        if (exceptionRecord.scannedDocuments == null) {
+            // ensure scanned documents not null
+            caseData.put(SCANNED_DOCUMENTS, scannedDocuments);
+            return;
+        }
+
+        List<String> exceptionRecordDcns = exceptionRecord.scannedDocuments
+            .stream()
+            .map(scannedDocument -> scannedDocument.controlNumber)
+            .collect(toList());
+
+        if (exceptionRecordDcns.isEmpty()) {
```

Comment:
DCN is mandatory this can never ben empty at this point.

Reply:
ok, also just wanted to be bulletproof